packages/theming/atlas/scripts/build.js

```diff
@@ -70,7 +70,8 @@ async function buildAndCopyAtlas(watchMode, destination) {
             }
         );
         console.log("Success", success);
```

Comment:
What about just having "Success" (string literal) without the large object log that usually happens on build?

Reply:
Good suggestion! I changed it.